src/components/TodoFilter/TodoFilter.tsx

```diff
@@ -0,0 +1,49 @@
+import classNames from 'classnames';
+import { Filter } from '../../types/Filter';
+
+type Props = {
+  filter: Filter;
+  onFilterChange: (newFilter: Filter) => void;
+};
+
+export const TodosFilter: React.FC<Props> = ({
+  filter,
+  onFilterChange = () => {},
```

Comment:
```suggestion
  onFilterChange,
```
default value is redundant - prop is required